lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -23,10 +23,19 @@ class SynchronizeLdm < BaseAction
 
         description 'LDM Update Preference'
         param :update_preference, instance_of(Type::UpdatePreferenceType), required: false
+
+        description 'Specifies whether to transfer computed attributes'
+        param :include_computed_attributes, instance_of(Type::BooleanType), required: false, default: true
       end
 
       class << self
         def call(params)
+          # set default value for include_computed_attributes
+          # (we won't have to do this after TMA-690)
+          include_ca = params.include_computed_attributes
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)